cmd/serve/nfs/handler.go

```diff
@@ -0,0 +1,70 @@
+//go:build unix
+// +build unix
+
+package nfs
+
+import (
+	"context"
+	"net"
+
+	"github.com/go-git/go-billy/v5"
+	"github.com/rclone/rclone/vfs"
+	"github.com/willscott/go-nfs"
+	nfshelper "github.com/willscott/go-nfs/helpers"
+)
+
+// NewBackendAuthHandler creates a handler for the provided filesystem
+func NewBackendAuthHandler(vfs *vfs.VFS) nfs.Handler {
+	return &BackendAuthHandler{vfs}
+}
+
+// BackendAuthHandler returns a NFS backing that exposes a given file system in response to all mount requests.
+type BackendAuthHandler struct {
+	vfs *vfs.VFS
+}
+
+// Mount backs Mount RPC Requests, allowing for access control policies.
+func (h *BackendAuthHandler) Mount(ctx context.Context, conn net.Conn, req nfs.MountRequest) (status nfs.MountStatus, hndl billy.Filesystem, auths []nfs.AuthFlavor) {
+	status = nfs.MountStatusOk
+	hndl = &FS{vfs: h.vfs}
```

Comment:
Why not use the full name (handle) instead of the abbreviation?

Reply:
I copy pasted this code.